bundles/admin/admin-layereditor/view/AdminLayerForm/VisualizationTabPane/StyleSelect.jsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
```

Comment:
Rename this to VectorStyleSelect (so match RasterStyle in #1416) so it's easier to figure out what style we are talking about.

Reply:
Or as it's only used in VectorStyle.jsx create a folder for VectorStyle and move this under that folder.